src/main/java/su/terrafirmagreg/core/common/data/items/PiglinDisguise.java

```diff
@@ -0,0 +1,27 @@
+package su.terrafirmagreg.core.common.data.items;
+
+import net.minecraft.tags.ItemTags;
+import net.minecraft.world.entity.EquipmentSlot;
+import net.minecraft.world.entity.LivingEntity;
+import net.minecraft.world.item.BlockItem;
+import net.minecraft.world.item.Item;
+import net.minecraft.world.item.ItemStack;
+import net.minecraft.world.level.block.Block;
+import org.jetbrains.annotations.Nullable;
+
+public class PiglinDisguise extends BlockItem {
+
+    public PiglinDisguise(Block block, Item.Properties properties) {
+        super(block, properties);
+    }
+
+    @Override
+    public EquipmentSlot getEquipmentSlot(ItemStack stack) {
+        return EquipmentSlot.HEAD;
+    }
+
+    @Override
+    public boolean makesPiglinsNeutral(ItemStack stack, @Nullable LivingEntity wearer) {
+        return stack.is(ItemTags.PIGLIN_LOVED);
```

Comment:
wow is this really all that's needed for making them friendly? I swear it was hardcoded to gold armor somewhere deep inside forge code